src/import-data/ImportDataDestination.test.ts

```diff
@@ -353,6 +353,120 @@ describe('ImportDataDestination', () => {
     expect(isNoticeShown).toBe(shouldShowNotice);
   });
 
+  it.each([
+    {
+      importRequest: { type: 'pfb', url: new URL('https://service.prod.anvil.gi.ucsc.edu/path/to/file.pfb') },
+      shouldShowAdditionalAccessControlNotice: true,
+    },
+    {
+      importRequest: { type: 'pfb', url: new URL('https://example.com/path/to/file.pfb') },
+      shouldShowAdditionalAccessControlNotice: false,
+    },
+  ] as {
+    importRequest: ImportRequest;
+    shouldShowAdditionalAccessControlNotice: boolean;
+  }[])(
+    'displays additional access controls message when importing protected data',
+    async ({ importRequest, shouldShowAdditionalAccessControlNotice }) => {
+      // Arrange
+      const user = userEvent.setup();
+
+      asMockedFn(canImportIntoWorkspace).mockImplementation(
+        (_importOptions: ImportOptions, _workspace: WorkspaceWrapper): boolean => {
+          return true;
+        }
+      );
```

Comment:
For what it's worth, this could also be written a bit more concisely with `mockReturnValue`.

```suggestion
      asMockedFn(canImportIntoWorkspace).mockReturnValue(true);
```

Reply:
One question about having to add this mock...I was surprised that I didn't get it 'for free' by the global mock at the very top of the file.  Does that surprise you too?  I was going to ask about it as a comment but you beat me to the punch! :D

Reply:
I think it's because previous tests change the mock behavior. The mock factory function initializes the mock, but it doesn't act like `beforeEach`.

Reply:
Oh jeez, good to know -- there's no automated mock teardown _after_ tests?

Reply:
Mocks are automatically _cleared_, but not _reset_ or _restored_ after each test.

- https://jestjs.io/docs/configuration#clearmocks-boolean
- https://jestjs.io/docs/configuration#resetmocks-boolean
- https://jestjs.io/docs/configuration#restoremocks-boolean